### ISSUE TITLE
SEED Lab development: password cracking

### ISSUE THREAD

**Issue by cole-wustl** [seed-labs/seed-labs#31]
My co-instructor and I have a student group in an Intro to Security class who, for a class project, are designing a password-cracking lab after the model of the SEED labs we've done in class.  Is this a topic of potential interest for a SEED Lab?  If so, what would be the best way to collaborate and see whether the SEED Project would be interested in including their work?  Happy to connect the group and discuss midstream or wait until the assignment is fully functional and present it then for a look.

**Comment by kevin-w-du** [comment]
Yes, this is also a good topic. See my comments on Issue #30 regarding how to collaborate.

**Comment by cole-wustl** [comment]
Great, thank you!
